__tests__/frontend/PlaylistContainer-test.js

```diff
@@ -91,10 +91,14 @@ describe('Check video navigation', () => {
     expect(videos.length).not.toBe(0)
 
     fireEvent.press(videos[0])
-    const firstVideo = VIDEOS_BY_PLAYLIST_RESPONSE.medias.find((v) => v.index === 0)
-    const expectedVideoUrl = FILE_SERVER_BASE_URL + firstVideo.video.video1Path
+    // const firstVideo = VIDEOS_BY_PLAYLIST_RESPONSE.medias.find((v) => v.index === 0)
+    // const expectedVideoUrl = FILE_SERVER_BASE_URL + firstVideo.video.video1Path
 
     expect(mockNaivgator.push).toHaveBeenCalled()
-    expect(mockNaivgator.push).toHaveBeenCalledWith(STACK_SCREENS.VIDEO, { url: expectedVideoUrl })
+    // expect(mockNaivgator.push).toHaveBeenCalledWith(STACK_SCREENS.VIDEO, { url: expectedVideoUrl })
```

Comment:
Can you fix this test to run with your changes. The second parameter is the argument we expect from the callback. So we probably would expecte something like `{ videos: videos, index: 0 }`

Reply:
I rewrite this in line 99. I'm not sure why the online editor not showing it lol

Reply:
Ok cool, can you just delete the commented out code then?